<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectPropertyEditorForm</class>
 <widget class="QWidget" name="ObjectPropertyEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Свойства объекта</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="shortName_lineEdit">
        <property name="placeholderText">
         <string>Отображаемое имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_lineEdit">
        <property name="placeholderText">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="description_plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="property_tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab_colors">
         <attribute name="title">
          <string>Цвета</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Основной цвет:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="mainColor_label">
              <property name="text">
               <string>#000000</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="selectMainColor_pushButton">
              <property name="text">
               <string>Выбрать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Цвет фона:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="bgrColor_label">
              <property name="text">
               <string>#000000</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="selectBgrColor_pushButton">
              <property name="text">
               <string>Выбрать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Цвет выбора:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="selectedColor_label">
              <property name="text">
               <string>#000000</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="selectSelectionColor_pushButton">
              <property name="text">
               <string>Выбрать</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>111</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_icon">
         <attribute name="title">
          <string>Иконка</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="iconPreview_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="iconPath_lineEdit">
              <property name="placeholderText">
               <string>Путь до файла...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="selectIcon_pushButton">
              <property name="text">
               <string>Выбрать...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="accept_pushButton">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_pushButton">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
